<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: Thursday, 17 July 2025 7:30:48</msg><msg code="0" type="0">SQL Filter: Code IN (13, 409960)</msg><msg code="-2147467259" type="100">﻿Traceback (most recent call last):
﻿  File "E:\git\LevellingThesis\Code\AdjsutMeasurements.py", line 134, in &lt;module&gt;
﻿    for row in cursor:
﻿RuntimeError: Cannot find field 'PointCode'
</msg><msg code="0" type="0">Failed script Network Adjustment...</msg><msg code="-2147467259" type="100">Failed to execute (NetworkAdjustment).</msg><msg code="0" type="3">Failed at Thursday, 17 July 2025 7:30:58 (Elapsed Time: 10.19 seconds)</msg></messages>